tests/acceptance/features/apiSharingNg/enableDisableShareSync.feature

```diff
@@ -297,3 +297,43 @@ Feature:  enable or disable sync of incoming shares
       | resource      |
       | textfile0.txt |
       | FolderToShare |
+
+  @issue-8724
+  Scenario: trying to enable share sync using non-existing remote item-id
```

Comment:
```suggestion
  Scenario: trying to enable share sync of non-existing resource
```
Nothing is shared and nothing is created in the test. so IMO this title would sound good